tests/test_optimizer_random_dag.py

```diff
@@ -140,6 +153,9 @@ def compare_optimization_results(dag: sky.Dag, minimize_cost: bool):
         objective = sky.Optimizer._compute_total_time(dag.get_graph(),
                                                       dag.tasks, optimizer_plan)
 
+    print('=== optimizer plan ===', file=sys.stderr)
+    print(optimizer_plan, file=sys.stderr)
+    print('=== brute force ===', file=sys.stderr)
```

Comment:
Is it intended? Are those debug codes that forgot to delete?

Reply:
It will be included in another PR #2976. : )